archivist/events.py

```diff
@@ -107,7 +216,20 @@ def read_by_signature(
         attrs=None,
         asset_attrs=None,
     ):
-        """docstring"""
+        """Read event by signature.
+
+        Reads event that meets criteria. Only one event is expected.
+
+        Args:
+            asset_id (str): optional asset identity e.g. assets/xxxxxxxxxxxxxxxxxxxxxxxxxx
+            props (dict): e.g. {"tracked": "TRACKED" }
+            attrs (dict): e.g. {"arc_display_type": "door" }
```

Comment:
Again event display_type should be ‘open’

Reply:
fixed